src/Klsandbox/BillplzRoute/Services/BillplzResponseManager.php

```diff
@@ -52,7 +52,7 @@ public function getBill($bill_id)
         return $bill;
     }
 
-    public function createBill($data)
+    public function createBill($data, $billplzKey)
```

Comment:
There is collection_id that needs to be set as well.

Where are the consumers for this?